functions/Directory.Build.props

```diff
@@ -0,0 +1,7 @@
+<Project>
+  <PropertyGroup>
+    <LangVersion>latest</LangVersion>
+    <Nullable>enable</Nullable>
```

Comment:
Using MSBuild's `Directory.Build.props` to set these props for all projects in the solution.